examples/app-setup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -e
```

Comment:
If we standardise on shell scripts for testing example app, then it's good to add also an unbound symbol check, to be future-proof.  See e.g. [Invenio scripts](https://github.com/inveniosoftware/invenio/blob/master/scripts/populate-instance.sh#L98-L100) where I'm using:

```
# quit on errors:
set -o errexit

# quit on unbound symbols:
set -o nounset
```

But let me reply in the other shell-scripting-vs-Python-scripting thread...